workers/generic-worker/mounts.go

```diff
@@ -714,24 +714,38 @@ func ensureCached(fsContent FSContent, taskMount *TaskMount) (file string, err e
 	return
 }
 
-func extract(fsContent FSContent, format string, dir string, taskMount *TaskMount) error {
-	cacheFile, err := ensureCached(fsContent, taskMount)
+func extract(fsContent FSContent, format string, dir string, taskMount *TaskMount) (err error) {
+	var cacheFile string
+	cacheFile, err = ensureCached(fsContent, taskMount)
 	if err != nil {
 		log.Printf("Could not cache content: %v", err)
 		return err
```

Comment:
nit: `return` would be sufficient here, but it doesn't really matter

Reply:
😅 I meant to remove these, just missed one! Thank you